### PR TITLE
Add typos found in various software projects

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -234,6 +234,7 @@ aboved->above
 abovemtioned->abovementioned
 aboves->above
 abovmentioned->abovementioned
+abpout->about
```
Comment (TheGiraffe3) [inline codespell_lib/data/dictionary.txt:237]:
```suggestion
abpout->about, a pout,
```
Possibly?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7104,6 +7112,7 @@ assemalate->assimilate
 assemalated->assimilated
 assemalates->assimilates
 assemalating->assimilating
+assembalge->assembladge
```
Comment (TheGiraffe3) [inline codespell_lib/data/dictionary.txt:7115]:
```suggestion
assembalge->assemblage
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -23865,6 +23882,7 @@ elelmentary->elementary
 elelments->elements
 elemant->element
 elemantary->elementary
+elemants->elements
```
Comment (TheGiraffe3) [inline codespell_lib/data/dictionary.txt:23885]: I don't suppose elephants should be added here as a possible fix?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -29269,6 +29291,9 @@ furtermore->furthermore
 furtest->furthest
 furthe->further
 furthemore->furthermore
+furthere->further
+furtheremor->furthermore
+furtheremore->furthermore
```
Comment (TheGiraffe3) [inline codespell_lib/data/dictionary.txt:29296]: How about 
furthermor->furthermore
?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:7115]: Done. I also added 
`assembladge->assemblage`

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:23885]: I think that's a stretch

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:29296]: furthermor->furthermore exists already in dictionary.txt

**Review by TheGiraffe3 — APPROVED** [review]

**Comment by luzpaz** [comment]
Sorry, I resolved the other open review requests and it changed the approved to pending again.

**Review by TheGiraffe3 — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -63317,6 +63356,7 @@ weilded->wielded
 weill->will
 weired->weird
 weitght->weight
+weith->weight, with,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:63359]:
```suggestion
weith->weight, weigh, with,
```

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**